src/js/containers/account/visualizations/AccountTimeVisualizationContainer.jsx

```diff
@@ -153,8 +223,93 @@ export class AccountTimeVisualizationSectionContainer extends React.Component {
         const xSeries = [];
         const ySeries = [];
         const allY = [];
-        if (this.state.hasFilteredObligated) {
-            // In progress
+        const hasFilteredObligated = (this.props.reduxFilters.objectClass.count() > 0)
```

Comment:
Made hasFilteredObligated a state variable & passed down to child components as a prop